packages/trader/src/AppV2/app.tsx

```diff
@@ -5,7 +5,7 @@ import type { TCoreStores } from '@deriv/stores/types';
 import { routes } from '@deriv/shared';
 import ModulesProvider from 'Stores/Providers/modules-providers';
 import TraderProviders from '../trader-providers';
-import { ReportsStoreProvider } from '../../../reports/src/Stores/useReportsStores';
+import { ReportsStoreProvider } from '@deriv/reports/src/Stores/useReportsStores';
```

Comment:
This looks unintentional. Please revert this.

Reply:
Done